### PR TITLE
Update check-docstyle.sh script to check all files

### PR DESCRIPTION
**Pull request by tisnik**
Previous status: https://fabric8-analytics.github.io/dashboard/repository_fabric8-analytics-jobs.html

This change fix the problem with two source files that were not checked.

### CODE REVIEW

File: check-docstyle.sh [hunk 1]
```diff
@@ -46,6 +47,32 @@ do
 done
 
 
+echo
+echo "----------------------------------------------------"
+echo "Checking documentation strings in the following files"
+echo $separate_files
```
Comment (pkajaba) [inline check-docstyle.sh]: can't you extract this into the function, which would take a list of files?

Comment (tisnik) [reply check-docstyle.sh]: Yeah I was thinking about refactoring this ugly part. Let's try this: )

Comment (tisnik) [reply check-docstyle.sh]: WDYT?

Comment (pkajaba) [reply check-docstyle.sh]: Good job, thanks.

**Pull request merged**